app/javascript/components/widgets/twitter/tweet_content.jsx

```diff
@@ -86,10 +86,10 @@ TweetText.propTypes = {
 };
 
 const imageLayout = images => {
-  return images.map(image => {
+  return images.map((image, index) => {
```

Comment:
What does adding the index, or unique key, do in this situation to eliminate the error?

Reply:
https://reactjs.org/docs/lists-and-keys.html - error 404 on the page this link should lead to

Reply:
nevermind, it just works now... idk